python/mscclpp/nvls_py.cpp

```diff
@@ -34,5 +34,9 @@ void register_nvls(nb::module_& m) {
       .def("get_multicast_min_granularity", &NvlsConnection::getMultiCastMinGranularity);
 
   m.def("connect_nvls_collective", &connectNvlsCollective, nb::arg("communicator"), nb::arg("allRanks"),
+#if (USE_NVLS)
```

Comment:
I think this issue is resolved by previous PR. We can revert this file

Reply:
Fixed.